src/bind.ts

```diff
@@ -1,6 +1,6 @@
-import { UnbindFn, Binding } from './types';
+import { UnbindFn, Binding, PossibleEventType } from './types';
 
-export function bind<Target extends EventTarget, Type extends string>(
+export function bind<Target extends EventTarget, Type extends PossibleEventType<keyof Target>>(
```

Comment:
this is magic 👏

Reply:
A true magic was probably here 😅 : https://github.com/alexreardon/bind-event-listener/pull/45/files